packages/fxa-settings/src/pages/ConfirmResetPassword/en.ftl

```diff
@@ -0,0 +1,10 @@
+## Confirm Reset Password Component
+
+# Second step of password reset flow for Firefox accounts
+# Header confirming that a password reset email has been sent to the user's email address
+confirm-pw-reset-header = Reset email sent
+
+# Instructions to continue the password reset process
+# ${email} is the email entered by the user and where the password reset instructions were sent
+# "follow" is prefered to "click" or "tap" for better accessibility
+confirm-pw-reset-instructions = Follow the link emailed to ${email} within the next hour to create a new password.
```

Comment:
The variable formatting is not one that I expect to see in Fluent…

```suggestion
confirm-pw-reset-instructions = Follow the link emailed to {$email} within the next hour to create a new password.
```

Reply:
Thank you! I've corrected the variable.